libs/ui-toolkit/src/utils/intent.tsx

```diff
@@ -19,7 +19,8 @@ export const getIntentShadow = (intent?: Intent) => {
 
 export const getVariantBackground = (variant?: Intent) => {
```

Comment:
**nitpick (non-blocking)**: Nittiest nitpick - the function name implies it will only control the background, while the classnames actually also now control text colour. I can live with this.

Reply:
Cheers @edd , 'tis a good point. I will leave it as this for now here, but change it in the bigger ui-toolkit updates ticket as that already changes this funtion name.